src/hapi/pipelines/database/operational_presence.py

```diff
@@ -149,46 +158,23 @@ def populate(self, debug=False):
                                             org_type_name
                                         )
                                     )
-                        if org_type_name and not org_type_code:
-                            add_missing_value_message(
-                                errors, dataset_name, "org type", org_type_name
-                            )
-                        self._org.add_or_match_org(
-                            acronym=org_acronym,
-                            org_name=org_name,
-                            org_type=org_type_code,
-                        )
-                        org_acronym, org_name, org_type = self._org.data[
-                            (
-                                normalise(org_acronym),
-                                normalise(org_name),
-                            )
-                        ]
-                        sector_code = self._sector.get_sector_code(sector_orig)
-                        if debug:
-                            debug_row = {
-                                "location": country_code,
-                                "org_name_orig": org_name_orig,
-                                "org_acronym_orig": org_acronym_orig,
-                                "org_type_orig": org_type_orig,
-                                "sector_orig": sector_orig,
-                                "org_name": org_name,
-                                "org_acronym": org_acronym,
-                                "org_type": org_type_code,
-                                "sector": sector_code,
-                            }
-                            if debug_row in debug_rows:
-                                continue
-                            debug_rows.append(debug_row)
-                            continue
+                                    if not org_type_code:
+                                        add_missing_value_message(
+                                            errors,
+                                            dataset_name,
+                                            "org type",
+                                            org_type_name,
+                                        )
 
-                        if not sector_code:
-                            add_missing_value_message(
-                                errors, dataset_name, "sector", sector_orig
-                            )
-                            continue
+                        # * Org matching
+                        org_acronym, org_name, _ = self._org.add_or_match_org(
```

Comment:
I think you'll need to return org type as well - because we have a uniqueness requirement on org name and acronym, we can't have orgs with the same name and acronym with different org types.  There's a todo in org.py to address this.

Reply:
Th org type is in the tuple put into the org `data` member variable used to populate that table. I haven't returned it here because it isn't used in the operational presence table or other lines below. (It was probably in the `if debug` section before).